lib/broadway_rabbitmq/telemetry.ex

```diff
@@ -0,0 +1,30 @@
+defmodule BroadwayRabbitMQ.Telemetry do
+  @moduledoc false
+
+  # TODO: remove when telemetry comes out with a release that supports telemetry:span/3. The
+  # implementation here is basically taken straigh out of
+  # https://github.com/beam-telemetry/telemetry/blob/53fbf8abdebea0ff0d9de9bb09740c578533d479/src/telemetry.erl#L258
```

Comment:
v0.4.2 supports span. Depend on it like this: `>= 0.4.2 and < 2.0.0`.

Reply:
Oh nice, I tried out the latest version and it didn't have span. I completely missed that :\ Updated!